src/app/pitwall/dashboard/[id]/page.tsx

```diff
@@ -1,59 +1,29 @@
 "use client";
 
-import { DashboardCard } from "./components/dashboard-card";
+import Dashboard from "./components/dashboard";
 import "./style.css";
-import { Conditions } from "@/components/conditions/conditions";
 import PitwallSession from "@/components/connection/pitwall-session";
-import { Standings } from "@/components/standings/standings";
-import { TrackMap } from "@/components/trackmap/trackmap";
-import GridLayout, { ItemCallback, WidthProvider } from "react-grid-layout";
-
-const ResponsiveGridLayout = WidthProvider(GridLayout);
+import PitwallSessionMock from "@/components/connection/pitwall-session-mock";
+import { MOCKING } from "@/config/site";
 
 export default function Page({ params }: { params: { id: string } }) {
   const pitboxSessionId = params.id;
 
-  // #endregion
-  const onResizeStopped: ItemCallback = (layout, oldItem, newItem) => {
-    console.log("Grid-RESIZE", newItem.i);
+  const pitwallSession = () => {
+    if (MOCKING) {
+      return (
+        <PitwallSessionMock pitwallSessionId={pitboxSessionId}>
+          <Dashboard />
+        </PitwallSessionMock>
+      );
+    } else {
+      return (
+        <PitwallSession pitwallSessionId={pitboxSessionId}>
+          <Dashboard />
+        </PitwallSession>
```

Comment:
Nice, love how clean this is. Even if the mock data is static, still a step forward!

Reply:
Yes, I'm not yet fully happy with it. As it also means that we have the mock in the final app. I will try to dig a bit deeper and find a nice way to inject the mock on startup.